modules/engine/src/model/model.ts

```diff
@@ -344,7 +344,18 @@ export class Model {
    */
   setShaderModuleProps(props: Record<string, any>): void {
     const uniforms = this._getModuleUniforms(props);
+
+    // Extract textures & framebuffers set by the modules
+    // TODO better way to extract bindings
+    const keys = Object.keys(uniforms).filter(k => uniforms[k].constructor.name.includes('WEBGL'));
```

Comment:
I suppose anything that is not a number, boolean or array would qualify. Also, we may soon drop uniforms altogether, then there is only bindings.